cfe_internal/enterprise/federation/federation.cf

```diff
@@ -6,7 +6,7 @@ bundle common federation_file_control
 
 body file control
 {
-        inputs => { @(federation_file_control.inputs) };
+#        inputs => { @(federation_file_control.inputs) };
```

Comment:
So we are relying on this magically being loaded before this file is processed? Also, this breaks running `cf-promises` on the file, right?

Reply:
1) I don't know if the federation files are included elsewhere. So maybe we rely on this.
2) maybe not break cf-promises, but definately would break the abiltity to run cf-agent with this as an entry.

Reply:
Shouldn't we also remove the bundle common above which defines the "inputs" var?

Reply:
Yeah, if we redact it we can. Unless someone thinks declaring the dependancies just for documentation is good